docs/tutorial/part-zero/index.md

```diff
@@ -1,235 +1,235 @@
 
-### Learn more about the command line
+### Ulteriori informazioni sulla riga di comando
 
-For a great introduction to using the command line, check out [**Codecademy’s Command Line tutorial**](https://www.codecademy.com/courses/learn-the-command-line/lessons/navigation/exercises/your-first-command) for Mac and Linux users, and [**this tutorial**](https://www.computerhope.com/issues/chusedos.htm) for Windows users. Even if you are a Windows user, the first page of the Codecademy tutorial is a valuable read. It explains what the command line is, not just how to interface with it.
+Per un'ottima introduzione all'uso della riga di comando, dai un'occhiata al [**Tutorial di Codecademy, Command Line**](https://www.codecademy.com/courses/learn-the-command-line/lessons/navigation/exercises/your-first-command) per utenti Mac e Linux e [**questo tutorial**](https://www.computerhope.com/issues/chusedos.htm) per utenti Windows. Anche se sei un utente Windows, la prima pagina del tutorial Codecademy è una lettura preziosa. Spiega cos'è la riga di comando, non solo come interfacciarsi con essa.
 
-### Learn more about npm
+### Ulteriori informazioni su npm
 
-npm is a JavaScript package manager. A package is a module of code that you can choose to include in your projects. If you just downloaded and installed Node.js, npm was installed with it!
+npm è un gestore di pacchetti JavaScript. Un pacchetto è un modulo di codice che puoi scegliere di includere nei tuoi progetti. Se hai appena scaricato e installato Node.js, npm è stato installato con esso!
 
-npm has three distinct components: the npm website, the npm registry, and the npm command line interface (CLI).
+npm ha tre componenti distinti: il sito Web di npm, il registro di npm e l'interfaccia a riga di comando (CLI) di npm.
 
-- On the npm website, you can browse what JavaScript packages are available in the npm registry.
-- The npm registry is a large database of information about JavaScript packages available on npm.
-- Once you’ve identified a package you want, you can use the npm CLI to install it in your project or globally (like other CLI tools). The npm CLI is what talks to the registry — you generally only interact with the npm website or the npm CLI.
+- Sul sito web di npm, puoi esplorare quali pacchetti JavaScript sono disponibili nel registro di npm.
+- Il registro npm è un ampio database di informazioni sui pacchetti JavaScript disponibili su npm.
+- Una volta identificato un pacchetto che desideri, puoi utilizzare la CLI di npm per installarlo nel tuo progetto o globalmente (come altri strumenti CLI). La CLI di npm è la cos che comunica con il registro: generalmente interagisci solo con il sito Web di npm o con la CLI di npm.
```

Comment:
typo in "è la cos"

(proviamo a cambiarlo con "ciò" o "quello"? "la cosa" fa molto fantastici quattro)